engine_test.go

```diff
@@ -293,3 +294,27 @@ func newEngine(t *testing.T) *sqle.Engine {
 
 	return e
 }
+
+const expectedTree = `Offset(2)
+ └─ Limit(5)
+     └─ Project(t.foo, bar.baz)
+         └─ Filter(foo > qux)
+             └─ InnerJoin(foo = baz)
+                 ├─ TableAlias(t)
+                 │   └─ UnresolvedTable(tbl)
+                 └─ UnresolvedTable(bar)
+`
+
+func TestPrintTree(t *testing.T) {
```

Comment:
Should be this test on parse package?

Reply:
I'm only using parse to not create the tree by hand. Since it's kind of an integration test checking some nodes return the correct output, I thought of putting it on the engine test